routing.rst

```diff
@@ -1805,6 +1805,77 @@ with a locale. This can be done by defining a different prefix for each locale
             ;
         };
 
+.. _stateless-routing:
+
+Stateless Routes
+----------------
+
```

Comment:
We're also missing  a reason to use this feature. What about adding a short sentence as introduction?
Something like:
```rst
Sometimes, when an HTTP response should be cached, it is important to ensure
that can happen.
```
?

Reply:
I'd rather add more details for the reason why session is involved: 
```
Sometimes, when an HTTP response should be cached, it is important to ensure
that can happen. However, whenever session is started during a request, Symfony
turns the response into a private non-cacheable response.
```

WDYT ?

Reply:
Sounds good to me!